src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -16,6 +16,7 @@ namespace NuGet.Services.Metadata.Catalog.Persistence
     public class AzureStorage : Storage
     {
         private readonly CloudBlobDirectory _directory;
+        private readonly int MaxCountOfBlobSnapshots = 30;
```

Comment:
We need to make this snapshot count configurable. Flat container may need a different number of snapshots from catalog.